crates/nu-data/src/primitive.rs

```diff
@@ -148,6 +153,12 @@ pub fn get_color_config() -> HashMap<String, Style> {
         "leading_trailing_space_bg".to_string(),
         Style::default().on(Color::RGB(128, 128, 128)),
     );
+    // FIXME:
+    // this is a total hack in order to use this hashmap for boolean values
+    // if use_ls_colors = true in the color_config then the style will be RGB(1,1,1)
+    // which is an oddball color. So, when we see this color assume it's true
+    // otherwise assume it's false. Initialize to RGB(1,2,3).
+    hm.insert("use_ls_colors".to_string(), Color::RGB(1, 2, 3).normal());
```

Comment:
I don't quite understand this part. Are you saying you can't put true/false in the config?

Reply:
tbh, this 'hack' is because using the config system is such a pain to get a value out of a section. `use_ls_colors` exists in `[color_config]`. So, instead of going through the hassle of trying to get an actual value, I decided to hack it into submission this way. 

What this means is if you put `use_ls_colors = true` in the config under the `[color_config]` section, then it works the way you'd expect it to. It currently doesn't check for `true`. It really just checks to see that a valid key exists in that section and if it does, set it to a value that I can check for somewhere else RGB(1,1,1).

The 'right' thing to do is read the value properly but it's such a pain. I just couldn't mess with it any longer.